addons/website/static/src/scss/website.scss

```diff
@@ -1951,6 +1951,19 @@ $ribbon-padding: 100px;
     z-index: $zindex-modal + 1; // Over the modal backdrop.
 }
 
+// Search
+
+input[type="search"]::-webkit-search-cancel-button  {
+    -webkit-appearance: none;
+    content: escape-svg($btn-close-bg);
+    height: $input-font-size;
+    cursor: pointer;
+}
```

Comment:
In fact, this is might not be enough: searchbar have two possible style "default" and "light" you can choose in the editor. When you are in "default" mode, it uses the "default" bg color, which you can set using the "theme" panel in the editor, in the input section... and it that case it might not be good.

Should not you not touch the btn-close-bg color at all and use the same color as the input color in all cases somehow? :thinking: 

Also: maybe this could be in web/bootstrap_review_frontend so it's also true for portal when website is not installed?